Source/AllModdingComponents/CompAnimated/CompAnimated.cs

```diff
@@ -15,34 +15,57 @@ public class CompAnimated : ThingComp
         public int MaxFrameIndexMoving => Props.movingFrames.Count();
         public int MaxFrameIndexStill => Props.stillFrames.Count();
 
-        public Pawn Animatee => parent as Pawn;
+        private static bool AsPawn(ThingWithComps pAnimatee, out Pawn pawn)
+        {
+            bool asPawn = pAnimatee is Pawn;
+            pawn = asPawn? (Pawn) pAnimatee : null;
+            return asPawn;
+        }
+        
+        public override void PostDraw()
```

Comment:
render over thing then not a pawn rather than use as base layer like the PawnGraphicSet does